components/SectionTextBackground/SectionTextBackground.tsx

```diff
@@ -0,0 +1,66 @@
+import { VariantProps } from "../../dataStructures/VariantProps/VariantProps";
+import styled from "styled-components";
+
+const StyledSectionTextBackground = styled.section<VariantProps>`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: ${({ variant }) =>
+    variant === "dark" ? "var(--me-color-gray-500)" : "var(--me-color-white)"};
+  color: ${({ variant }) =>
+    variant === "dark" ? "var(--me-text-inverted)" : "var(--me-text-gray-700)"};
+`;
+
+const StyledHeadlineGroup = styled.div`
+  justify-content: flex-end;
+  align-self: flex-end;
+  padding-bottom: 5rem;
+  margin: 0.5rem;
+`;
+
+const StyledParagraph = styled.div`
```

Comment:
```suggestion
const StyledParagraph = styled.p`
```

A paragraph should be a **p** tag. **div** has no semantically meaning in HTML. Try to avoid using div if you can use a tag wich describes what the tag semantically stands for. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div?retiredLocale=de

In this case **p** describes a paragraph.